ui/app/pages/confirm-decrypt-message/confirm-decrypt-message.component.js

```diff
@@ -191,15 +192,15 @@ export default class ConfirmDecryptMessage extends Component {
         {this.renderAccountInfo()}
         <div className="request-decrypt-message__visual">
           <section>
-            {origin.icon ? (
+            {originMetadata.icon ? (
```

Comment:
Wait, does this need optional chaining?

Reply:
Hah, good catch! This was the mistake I had just mentioned finding - it has been fixed